src/components/_banner.scss

```diff
@@ -0,0 +1,65 @@
+@use "../design-tokens/color";
+@use "../design-tokens/typography";
+
+@mixin BannerTheme($color: color.$color-utilitiesGray, $iconColor: color.$color-blue) {
+  border-left: 0;
+  
+  &::before {
+    position: absolute;
+    top: -1px;
+    left: 0;
+    width: 3px;
+    height: 100%;
+    background-color: $color;
+    border: 1px solid $color;
+    border-radius: 4px 0 0 4px;
+    content: '';
+  }
+
+  .Manifold__BannerIcon {
+    svg {
+      fill: $iconColor;
+    }
+  }
+}
+
+@mixin Manifold__Banner {
+  position: relative;
+  display: flex;
+  box-sizing: border-box;
+  width: 100%;
+  padding: 10px 16px;
+  font-weight: 400;
+  font-size: typography.$typography-body-fontSize;
+  font-family: typography.$typography-body-fontFamily;
+  line-height: 1.25;
+  background: color.$color-white;
+  border-color: color.$color-utilitiesGray;
+  border-style: solid;
+  border-width: 1px;
+  border-radius: 4px;
+  @include BannerTheme();
+}
+
+@mixin Manifold__Banner--Error {
+  @include BannerTheme(color.$color-red, color.$color-red);
+}
+
+@mixin Manifold__Banner--Warning {
+  @include BannerTheme(color.$color-orange, color.$color-orange);
+}
+
+@mixin Manifold__Banner--Success {
+  @include BannerTheme(color.$color-green, color.$color-green);
+}
+
+@mixin Manifold__BannerIcon {
+  width: 20px;
+  height: 20px;
+  margin-right: 8px;
```

Comment:
I think this will work, but it never hurts to add `flex-grow: 0; flex-shrink: 0;` here too, to make sure that it won’t stretch

Also, for the text wrapper, that should probably get `flex-grow: 1; flex-shrink: 1;`